ngraph/frontend/onnx/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+add_subdirectory(onnx_common)
+add_subdirectory(onnx_import)
+if (NGRAPH_ONNX_EDITOR_ENABLE)
+    add_subdirectory(onnx_editor)
+    add_subdirectory(frontend)
```

Comment:
We need to change the logic here, it's not intuitive that `NGRAPH_ONNX_EDITOR_ENABLE` controls building of ONNX Frontend. 
ONNX Frontend should have it's own option to turn it on, which turns on building of importer and editor if set.